packages/cf-component-input/example/basic/component.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Input from 'cf-component-input';
+import { Input } from 'cf-component-input';
```

Comment:
[That is my understanding, yes :)](https://github.com/cloudflare/cf-ui/pull/285#discussion_r124845361)